pelican/tools/templates/tasks.py.jinja2

```diff
@@ -112,6 +112,9 @@ def livereload(c):
     for extension in static_file_extensions:
         static_file = '{0}/static/**/*{1}'.format(theme_path, extension)
         server.watch(static_file, lambda: build(c))
+    # Fire web browser at the URL
```

Comment:
Suggestion for re-wording this comment:  `# Open site in new web browser tab/window`